Sources/ProjectAutomation/Tuist.swift

```diff
@@ -26,15 +26,15 @@ public enum Tuist {
 
     /// Loads and returns the graph at the given path.
     /// - parameter path: the path which graph should be loaded. If nil, the current path is used.
-    public static func graph(at path: String? = nil) throws -> Graph {
+    public static func graph(at path: String? = nil, format: String = "json") throws -> Graph {
```

Comment:
```suggestion
    public static func graph(at path: String? = nil) throws -> Graph {
```

Reply:
Hi @fortmarek, thank you for a quick feedback on this PR. You are correct about legacyJSON but the idea was that we could specify it when we call `Tuist.graph(format: "legacyJSON")` at the call site instead. I am okay if you think we should hardcode the format because this framework is being deprecated but are there any downsides of taking a parameter instead?

Reply:
> You are correct about legacyJSON but the idea was that we could specify it when we call Tuist.graph(format: "legacyJSON") at the call site instead. I am okay if you think we should hardcode the format because this framework is being deprecated but are there any downsides of taking a parameter instead?

Given this framework is being deprecated, I'd rather keep the former behavior. Passing a format could easily make the command break as the decoding of the graph will fail, so any other format than `legacyJSON` will result an invalid execution.

Reply:
Gotcha, thanks for the additional context 👍